src/verify_cert.rs

```diff
@@ -324,6 +405,39 @@ fn check_eku(
     }
 }
 
+type KeyUsageBit = usize;
+
+// https://www.rfc-editor.org/rfc/rfc5280#section-4.2.1.3
+// const KU_DIGITAL_SIGNATURE: KeyUsageBit = 0;
+// const KU_CONTENT_COMMITMENT: KeyUsageBit = 1;
+// const KU_KEY_ENCIPHERMENT: KeyUsageBit = 2;
+// const KU_DATA_ENCIPHERMENT: KeyUsageBit = 3;
+// const KU_KEY_AGREEMENT: KeyUsageBit = 4;
+// const KU_KEY_CERT_SIGN: KeyUsageBit = 5;
+const KU_CRL_SIGN: KeyUsageBit = 6;
+// const KU_ENCIPHER_ONLY: KeyUsageBit = 7;
+// const KU_DECIPHER_ONLY: KeyUsageBit = 8;
```

Comment:
I personally thought it was nicer to have the unused bits represented here for context but if folks disagree I can remove the commented out bits, leaving only `KU_CRL_SIGN`.